<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDisplaySettings</class>
 <widget class="QDialog" name="DialogDisplaySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>323</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>323</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>323</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="displaySettingsBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Node Subset Settings</string>
   </property>
   <widget class="QCheckBox" name="showIncompatibleCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>53</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Pairings/NDDs with No Compatibilities</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>73</y>
      <width>402</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="displaySettingsGridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="praCheckBox">
       <property name="text">
        <string>Display Pairings with Candidates in PRA Range</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="praSpinBoxRight">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="praSpinBoxLeft">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="showExcludedCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Pairings/NDDs Excluded From Match Runs</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="nodeDisplayModeLabel">
        <property name="text">
         <string>Donor Display Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="nodeDisplayModeComboBox">
        <item>
         <property name="text">
          <string>Display Multiple Paired Donors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Display All Paired Donors</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Display No Paired Donors</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QVBoxLayout" name="displayVerticalLayout">
      <item>
       <widget class="QRadioButton" name="allNodesButton">
        <property name="text">
         <string>Display All Pairings/NDDs</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showArrangementsRadioButton">
        <property name="text">
         <string>Display Only Pairings/NDDs Involved in Transplant Arrangements</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="showSolutionsRadioButton">
        <property name="text">
         <string>Display Only Pairings/NDDs Involved in Solutions</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="nodeSubsetButton">
        <property name="text">
         <string>Display Pairings/NDDs Subset...</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDisplaySettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDisplaySettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nodeSubsetButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>displaySettingsBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>praSpinBoxLeft</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>praSpinBoxRight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
